packages/web-app-files/src/index.js

```diff
@@ -300,5 +318,14 @@ export default {
   routes,
   navItems,
   quickActions,
-  translations
+  translations,
+  mounted({ router: runtimeRouter, store: runtimeStore }) {
```

Comment:
This looks like a rebase-gone-wrong since it's already in latest master. @paulcod3 could you check and confirm? :)

Reply:
yes i think something went wrong when i was combining commits, should now be fixed